components/navigation/JobPostItem.js

```diff
@@ -1,46 +1,92 @@
+import { join } from "lodash";
+
 export default function JobPostItem({ job }) {
-  return (    
+  return (
     <li>
       <a href={job.slug} className="block hover:bg-gray-50">
         <div className="px-4 py-4 sm:px-6">
           <div className="flex items-center justify-between">
             <p className="text-sm font-medium text-indigo-600 truncate">
-              { job.title }
+              {job.title}
             </p>
+            <a href={job.company.url}>
+              <div className="ml-2 flex-shrink-0 flex">
+                <img src={job.company.logo_url} height="10px" width="20px" />
```

Comment:
Let's not hardcode styles inline. Instead of using `height` and `width` attributes use `className`. Also, the Postman logo has this proportion but others may not so we should only set the height.

Reply:
done !